pkg/apis/workload/v1alpha1/types.go

```diff
@@ -112,4 +112,8 @@ const (
 	// AnnotationSkipDefaultObjectCreation is the annotation key for an apiexport or apibinding indicating the other default resources
 	// has been created already. If the created default resource is deleted, it will not be recreated.
 	AnnotationSkipDefaultObjectCreation = "workload.kcp.dev/skip-default-object-creation"
+
+	// InternalSyncTargetPlacementAnnotationKey is a internal annotation key on placement API to mark the synctarget scheduled
+	// from this placement. The value is {location workspace}/{syncTarget name}
```

Comment:
Actually this must be the hash of the value. See @jmprusi's open PR.

Reply:
can be follow-up depending on which PR merges first.

Reply:
Let's merge this one first and I will rebase mine with those changes.